packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1001,6 +1001,8 @@ export class ContainerRuntime
 	 */
 	public readonly gcTombstoneEnforcementAllowed: boolean;
 
+	private readonly documentGuid: string;
```

Comment:
You should consider naming this to something like `telemetryDocumentId` since this is supposed to be used for telemetry only. Otherwise, its easy to confuse this with the concept of "documentId" that already exists.
Also, add a comment explaining what this is and what it should be used for.